Documentation/compatibility/winforms-accessibility-changes-471.md

```diff
@@ -101,11 +101,11 @@ __Improved Narrator support__
 - 
 
 
-### Affected APIs
+### Affected APIs not detectable via API analysis
```

Comment:
@merriemcgaw, I apologize -- I completely overlooked the list  of affected APIs. "not detectable via API analysis" should be removed. Otherwise, this looks good and is ready to merge.